### PR TITLE
Add new study: Searson_2004

### PR DESCRIPTION
**Pull request by ehwenk**
* no attempt to contact author, since dataset 19 years old
* data transcribed from tables and extracted from figures in manuscript

### CODE REVIEW

File: data/Searson_2004/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    ORCID: .na
+    additional_role: no contact
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      mutate(
+        location = "Hawkesbury campus",
+        vessel_diameter = 2*sqrt(`Vessel lumen area (um2 )`)/3.14159
```
Comment (yangsophieee) [inline data/Searson_2004/metadata.yml:29]: I think pi also has to be in the square root, like: 
```2*sqrt(`Vessel lumen area (um2 )`/3.14159)```

**Review by yangsophieee — APPROVED** [review]
Looks great!

File: data/Searson_2004/metadata.yml (hunk 1 above)
Comment (yangsophieee) [reply data/Searson_2004/metadata.yml]: Fixed but please check if it's correct

Comment (ehwenk) [reply data/Searson_2004/metadata.yml]: Yes, you're right, I had brackets in the wrong place

**Pull request merged**